### PR TITLE
[VENDOR] Set certifi minimum version to 2023.7.22 LRN-41302

### PR DESCRIPTION
**Pull request by markus-liang-lrn**
## Checklist

- [ ] Feature
- [ ] Bug
- [x] Security
- [x] Documentation

- [x] ChangeLog.md updated

- [ ] Tests added
- [ ] All testsuites passed
- [x] `make dist` completed successfully

### CODE REVIEW

**Comment by bhavya-shukla-lrn** [comment]
@markus-liang-lrn  Looks good to me

**Review by bhavya-shukla-lrn — APPROVED** [review]

**Review by bhavya-shukla-lrn — APPROVED** [review]

File: ChangeLog.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
 - Add Annotations API as a service
 
+### Security
+- Bump certifi minimum version to 2023.7.22
```
Comment (bhavya-shukla-lrn) [inline ChangeLog.md:13]: @markus-liang-lrn  Can you ask Edwin to review this Thanks?

Comment (TheRealEdDawson) [reply ChangeLog.md:13]: Hi there @bhavya-shukla-lrn and @markus-liang-lrn , I reviewed. Looks great, guys. Thanks for making this fix to the Python SDK!

**Review by TheRealEdDawson — APPROVED** [review]
Looks good to me. The changelog looks good also. Thanks guys.

**Pull request merged**